src/components/metadata/metadata-row.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import MetaDataList from './metadata-list';
+import MetaDataValue from './metadata-value';
+import './styles/metadata.css';
+
+/**
+ * Shows metadata label and a given single value, or a list of values, or child elements
+ */
+const MetaDataRow = ({
+  label,
+  property,
+  kind = 'text',
+  empty = '-',
+  visible = true,
+  inline = true,
+  commas = true,
+  children
+}) => {
+  const showElements = React.isValidElement(children);
+  const showList = !showElements && Array.isArray(children) && children.length;
+  const showValue = !showList && !showElements;
+
+  return (
+    visible && (
+      <div className="pipeline-metadata__row">
+        <h3 className="pipeline-metadata__label">{label}</h3>
```

Comment:
Yeah I was wondering this, I didn't think `dt / dd / dfn` were appropriate here initially since it's not a `term: definition` relationship, but looking closer at the spec it does seem like `dd` is intended for that.

Reply:
Yeah I had that thought too, but I think it should be relatively straightforward to change the styling from `display: flexbox` to `display: block` and use different margin/padding top/bottom for the different rows. Each component could be just something like 
```es6
return (
  <>
    <dt>{label}</dt>
    <dd>{value or whatever}</dd>
  </>
)
```

Reply:
Yeah I've managed to wrangle this to work for now then, assuming we allow `dd` to have any kind of children. 

Though it isn't ideal from a flexibility and reusability point of view and may cause problems in future. Just as an example of what I'm getting at, think about trying to add sub-components into this sidebar that didn't fit the `dt / dd` model. 

I could certainly use the `<Container>` prop pattern again here if needed, but depending on sidebar design, we'd need to potentially split out multiple `<dl>` lists. Doing this with dynamic rows, like sorting or hide / show based on type, it may get complicated to support this markup.